learn/configuration/settings.md

```diff
@@ -12,6 +12,7 @@ This page describes the **index-level settings** available in Meilisearch and ho
 | **[sortableAttributes](/learn/configuration/settings.md#sortable-attributes)**    | [Strings] | Attributes to use when [sorting](/learn/advanced/sorting.md) search results  | `[]`                                                                         |
 | **[stopWords](/learn/configuration/settings.md#stop-words)**                       | List of words ignored by Meilisearch when present in search queries              | `[]`                                                                                        |
 | **[synonyms](/learn/configuration/settings.md#synonyms)**                          | List of associated words treated similarly                                       | `{}`                                                                                        |
+| **[typoTolerance](/learn/configuration/settings.md#typo-tolerance)**               |Typo tolerance settings                                             | `{}`                                                                                        |
```

Comment:
It's tricky as well because most of the other settings are strings or arrays, but this one is an object. So we can't really use the same format as the other short descriptions in the table.

I tried to think of a descriptive phrase and came up with `Adjust conditions for typo tolerance or disable it on certain words or attributes`

However, I'm honestly wondering if it would be wiser to go with something generic. I can see this feature evolving quite a bit with new forms of customization. So something like this could be better for longevity: `Object used to customize typo tolerance behavior`.